Update repositories.json

N3xar opened the pull request:
Added Fallout 4 curated 2025 Modlist by N3xar

tr4wzified commented:
- Repository link needs to be in same format as the other links, raw link to the repo without the refs/heads part in there.
- Image size is too big, please convert it to webp.
- MachineURL should be named after your list. Maybe something like 'N3xarsFO4List' or whatever you pick. Just know that you shouldn't change it anymore after I merge this PR, so I would avoid putting the year in there.


Title is really long by the way. Not a rejection reason, but it's gonna get cut off in the Wabbajack UI.

N3xar commented:
Thank you, I relish your feedback and have made adjustments on every point:

- created a new repo to align with revision of name/title changes
- Adjusted link to align with other entries (sorry about that)
- Converted image to webp and updated the link accordingly
- MachineURL changed to fo4nexNG

I will expand on my readme on my site shortly - as I come to understand the process fully I'll be able to structure my documentation accordingly.

tr4wzified commented:
Can you resolve the conflicts with the other added lists? After that I can merge, but I'm out of the house atm and the mobile Git merging experience isn't great. Otherwise I'll get to this in the evening

N3xar commented:
Thank you. I've resolved the conflicts. The other two links look correct (compared to the other entries), so I just corrected the syntax for the JSON. I trust that's all that was needed from me?
